indico/modules/events/timetable/controllers/manage.py

```diff
@@ -112,3 +113,21 @@ def _process_PATCH(self):
             with track_time_changes():
                 update_timetable_entry(self.timetable_entry, updates)
         return jsonify()
+
+    def _process_DELETE(self):
+        """Delete a timetable entry"""
+        delete_timetable_entry(self.timetable_entry)
+
+
+class RHTimetableBalloon(RHTimetableREST):
+    def _process(self):
+        if not self.timetable_entry:
```

Comment:
Not needed; it's always set unless someone adds a rule to the blueprint that calls this without the entry id